plugins/scenario/hu.bme.mit.gamma.scenario.model/src/hu/bme/mit/gamma/scenario/model/reduction/SimpleScenarioGenerator.java

```diff
@@ -121,6 +124,17 @@ private void inlineExpressions(ScenarioDeclaration simple, ScenarioDeclaration b
 					}
 				}
 			}
+			if (decl instanceof VariableDeclaration) {
```

Comment:
Rename the abbreviated identifiers to the full words, e.g., `declaration`, `variable`...